docs/PublishingGuide.md

```diff
@@ -0,0 +1,67 @@
+*Read our [User Guide](UserGuide.md#customizing-the-analysis) for more information*
+1. To access your site, go to [the settings of your fork in GitHub, under **GitHub Pages** section](../../../../publish-RepoSense/settings), look for `Your site is published at [LINK]` <br/>
+*The changes made to the configuration files should trigger Travis-CI to generate your report. Otherwise, you follow the [next section](#keeping-your-site-up-to-date-with-your-code-contribution) to manually trigger a build*
+> It takes a few minutes for report generation. Meanwhile, you can monitor the progress live at [Travis-CI's Builds](https://travis-ci.org/dashboard/builds). <br/>
+  Try accessing your site again when a green tick appears beside your fork.  
+  
+### Keeping your site up-to-date with your code contribution
+
+[Travis-CI](https://travis-ci.org/) offers `Cron Jobs` in intervals of daily, weekly or monthly.
+
+1. Login to [Travis-CI](https://travis-ci.org/)
+1. Go to [your fork in Travis-CI](https://travis-ci.org/search/publish-RepoSense/), click on **More options** on the right then access **Settings**
+1. Under **Cron Jobs**, choose **master** for `Branch`, **Always run** for `Options` and pick an `Interval` of your choice; <br/>
+then click Add
+![Travis-CI Cron](images/publishingguide-cronsetting.jpg "Travis-CI Cron")
+
+Alternatively, you can manually trigger an update.
+
+1. Go to [your fork in Travis-CI](https://travis-ci.org/search/publish-RepoSense/), click on **More options** on the right then **Trigger build**
+1. In the pop up, click **Trigger custom build**
+
+### Specifying which version of RepoSense to use
+
+As RepoSense is being actively developed, its master branch is frequently updated with new features and fixes. <br/>
+For stablility or familiarity, you may want to use the release. <br/>
+
+#### Use our latest release (Stable)
+
+You can find the changelog of latest release [here](https://github.com/reposense/RepoSense/releases/latest).
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--release`. <br />
+```
+10   ./get-reposense.py --release
+```
+
+#### Use our master branch (Beta)
+
+You can find the list of commits to master branch [here](https://github.com/reposense/RepoSense/commits/master) and planned features/fixes [here](https://github.com/reposense/RepoSense/milestones).
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--master`. <br />
+```
+10   ./get-reposense.py --master
+```
+#### Use a specific version of release
+
+Be sure to read and understand [all breaking changes and bugs](https://github.com/reposense/RepoSense/releases) before proceeding.
+
+Edit line 10 of [run.sh](../../../../publish-RepoSense/edit/master/run.sh) to use `--tag TAG`. <br />
```

Comment:
... where `TAG` is the [target version of release](https://github.com/reposense/RepoSense/tags).